templates/base.html.twig

```diff
@@ -1,83 +1,69 @@
 <!DOCTYPE html>
-<html>
+<html lang="{{ lang() }}">
```

Comment:
Can we get dir= easily? If yes, add it in. If no, don't bother, but let's add that as a todo when we add proper language support with ULS.

Reply:
I've added it.